lib/screens/forgot_password.dart

```diff
@@ -9,68 +9,143 @@ class ForgotPassword extends StatefulWidget {
-        appBar: AppBar(),
-      ),
-      body: Container(
-        color: Colors.lightBlueAccent[100],
+  Container _emailField() {
+    return Container(
+      child: Form(
+        key: _formKey,
         child: Column(
-          crossAxisAlignment: CrossAxisAlignment.center,
           children: <Widget>[
+            Container(
+              padding: EdgeInsets.only(bottom: 20.0),
+              child: Text(
+                "If you've lost your password or wish to reset it, enter your email address and we'll send you instructions to create a new password.",
+                style: TextStyle(
+                    color: Colors.black,
+                    fontSize: 17.0,
+                    fontWeight: FontWeight.bold),
+                textAlign: TextAlign.center,
+              ),
+            ),
             TextFormField(
               keyboardType: TextInputType.emailAddress,
-              decoration: const InputDecoration(
-                hintText: 'Email',
+              decoration: InputDecoration(
+                filled: true,
+                fillColor: Color(0xF6F6F6F6),
+                enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Colors.grey[300]),
+                ),
+                prefixIcon: Icon(Icons.email),
+                labelText: 'Email',
               ),
-              onChanged: (value) {
+              onSaved: (value) {
                 setState(() {
                   _email = value.trim();
                 });
               },
             ),
+            SizedBox(height: 10.0),
             Visibility(
               visible: _exception == "" ? false : true,
               child: Text(
                 _exception,
                 style: TextStyle(color: Colors.red),
               ),
             ),
-            Padding(
-              padding: const EdgeInsets.symmetric(vertical: 16.0),
-              child: ElevatedButton(
-                onPressed: () => submit(),
-                child: Text('Send Request'),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Container _sendRequestButton(context) {
+    return Container(
+      width: 370.0,
+      height: 45.0,
+      child: RaisedButton(
+        color: Color(0xFF00ADB5),
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(20.0),
+        ),
+        onPressed: () => _submit(context),
+        child: Text(
+          'Reset My Password',
+          style: TextStyle(
+            color: Colors.white,
+            fontSize: 20.0,
+          ),
+        ),
+      ),
+    );
+  }
+
+  Future<dynamic> _showDialog(context) {
+    return showDialog(
+      context: context,
+      builder: (context) {
+        return AlertDialog(
+          title: Text("Request Sent"),
+          content: Text(
+              "Head over to your email and follow the instructions provided to reset your password."),
+          actions: <Widget>[
+            FlatButton(
```

Comment:
I am getting overflow for the forgot_password.dart 
![image](https://user-images.githubusercontent.com/36316432/111855237-e458aa80-88e0-11eb-8c11-05b5b802ec7e.png)

Reply:
Got the same on my Nexus 6 emulator